components/App.jsx

```diff
@@ -22,23 +22,25 @@ const App = () => {
 
     const [isSpeechBoxActive, setIsSpeechBoxActive] = useState(false);
     const [speechBoxData, isSpeechBoxData] = useState(dialogueObj.intro)
+
+    let location = useLocation();
     
     // I think this wants to be refactored to some other name
     const [isNav, setIsNav] = useState(false);
 
     // does this do way too much?
     const spriteClickHandler = () => {
-        if (isInitialSpriteClick) {
+        if(location.pathname === "/" && isInitialSpriteClick){
```

Comment:
Do we want to check if the path is root, or would we rather set isInitialSpriteClick to false if they are somewhere other than root?